packages/paste-website/src/pages/utilities/grid/index.mdx

```diff
@@ -0,0 +1,520 @@
+/>
+
+---
+
+<contentwrapper>
+
+<TableOfContents headings={props.data.mdx.headings} />
+
+<content>
+
+## Guidelines
+
+<Paragraph>{props.pageContext.frontmatter.description}</Paragraph>
+
+### About Grid
+
+Grid creates organized and consistent product layouts, by providing a standard for positioning elements. Grid speeds up the designer-to-developer workflow by improving the consistency of design implementations.
+
+Use a Grid to layout and align content by applying standard spacing between elements on a page or within a component.
+
+The standard desktop grid has 24-pixel gutters and can support up to 12 columns. The outside gutters are removed from the overall width of the grid.
+
+Grid is built using the [Flex component](/utilities/flex) and can be made responsive.
+
+## Examples
+
+#### Equal column widths
+
+Use Column to lay out equal-sized content, such as a feed of cards.
+
+<LivePreview scope={{Grid, Column, Card, Heading, Paragraph}} language="jsx">
+  {`<Grid gutter="space30">
+  <Column>
+    <Card padding="space70">
+      <Heading as="h4" variant="heading40">Card Heading</Heading>
+      <Paragraph>Chillwave fixie sustainable helvetica etsy.</Paragraph>
+    </Card>
+  </Column>
+  <Column>
+    <Card padding="space70">
+      <Heading as="h4" variant="heading40">Card Heading</Heading>
+      <Paragraph>Chillwave fixie sustainable helvetica etsy.</Paragraph>
+    </Card>
+  </Column>
+  <Column>
+    <Card padding="space70">
+      <Heading as="h4" variant="heading40">Card Heading</Heading>
+      <Paragraph>Chillwave fixie sustainable helvetica etsy.</Paragraph>
+    </Card>
+  </Column>
+</Grid>`}
+</LivePreview>
+
+#### Unequal column widths, using column span
+
+Use `span` to allow content to span multiple columns, such as a layout with a sidebar that takes up 1/4 of the screen and a content area that takes up 3/4 of the screen.
+
+<LivePreview scope={{Grid, Column, Card, Heading, Paragraph}} language="jsx">
+  {`<Grid gutter="space30">
+  <Column span={8}>
+    <Heading as="h4" variant="heading40">Content Heading</Heading>
+    <Paragraph>
+      Kale chips distillery authentic, portland etsy cloud bread vinyl gentrify drinking vinegar viral meh hot
+      chicken bitters fashion axe palo santo. Chillwave fixie sustainable <i>helvetica</i> etsy.
+    </Paragraph>
+  </Column>
+  <Column span={4}>
+    <Card padding="space70">
+      <Heading as="h4" variant="heading40">Card Heading</Heading>
+      <Paragraph>Chillwave fixie sustainable helvetica etsy.</Paragraph>
+    </Card>
+  </Column>
+</Grid>`}
+</LivePreview>
+
+<Callout variant="warning">
+  <CalloutTitle as="h4" variant="heading40">An Important Note About Span and Offset</CalloutTitle>
+  <CalloutText>When using the <strong>span</strong> or <strong>offset</strong> props, base your layout off a 12-column grid. For example, if you want to have two columns with one spanning 8 columns, one column would span 8 and the other would need to span 4.</CalloutText>
```

Comment:
`For example, if you want to have two columns with one spanning 8 columns, one column would span 8 and the other would need to span 4.` -> `For example, if you want to have two columns with one spanning twice as much as the other, then one column would span=8 and the other would span=4.`